### PR TITLE
Add a note that it is possible to chain XQuery Triggers together

### PR DESCRIPTION
**Pull request by adamretter**
As requested by @line-o for https://github.com/eXist-db/exist/pull/4280

### CODE REVIEW

**Review by line-o — APPROVED** [review]
That is a very helpful addition to the docs

**Review by joewiz — APPROVED** [review]

**Comment by joewiz** [comment]
Let's merge this once #4280 is merged.

**Pull request merged**
